node/resolvers/profile/services.ts

```diff
@@ -27,6 +32,16 @@ export function getProfile(context: Context, customFields?: string) {
     })
 }
 
+export function getPasswordLastUpdate(context: Context) {
+  const { request: { headers: { cookie } }, vtex: { account } } = context
+  const url = paths.getUser(account)
+  const parsedCookies = parse(cookie)
+  const userCookie: string = parsedCookies[`VtexIdclientAutCookie_${context.vtex.account}`]
```

Comment:
you already have account, just do ${account}

Reply:
you're right!